golden-tests/run-test.sh

```diff
@@ -1,9 +1,22 @@
 #!/usr/bin/env bash
 
-old_drv="$(nix-instantiate ./old-derivation/drv.nix)"
-new_drv="$(nix-instantiate ./new-derivation/drv.nix)"
-nix-diff $old_drv $new_drv --environment > /tmp/nix-diff-output
-diff ./expected-output /tmp/nix-diff-output
-code="$?"
-rm /tmp/nix-diff-output
-exit "$code"
+export OLD_DRV="$(nix-instantiate ./old-derivation/drv.nix)"
+export NEW_DRV="$(nix-instantiate ./new-derivation/drv.nix)"
+
+retval="0"
```

Comment:
Oh, btw, this looks like a perfect use-case for [`bats`](https://github.com/bats-core/bats-core). What do you think about reworking this script to use it instead?